src/components/ProjectWrapper/ProjectWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import AddProject from '@components/AddProject/AddProject'
+import { MyProjectSelect } from '@components/MyProjectSelection/MyProjectSelection'
+import React, { useState } from 'react'
+
+const ProjectWrapper: React.FC = (): JSX.Element => {
+  const [searchedProject, setSearchedProject] = useState<string>('')
+  return (
+    <div className="flex flex-col items-center h-full">

```

Comment:
by adding  "md:max-w-[467px] md:w-[35%] lg:w-[50%] w-full" we can improve a lot the responsiveness